### ISSUE TITLE
Gameplay für "Brauerei-Adventure mit Minispielen"

### ISSUE THREAD

**Issue by t-karcher** [t-karcher/Spiel-zum-Familientag#3]
Ich habe noch etwas Schwierigkeiten, mir das Spiel und dessen Ablauf vorzustellen. Hast du da schon konkrete Ideen? 

Ein erster Vorschlag: Wir teilen die Geschichte in mehrere Kapitel auf, die jeweils mit einem Intro beginnen, und in denen dann Aufgaben mit einem oder mehreren Mini-Spielen gelöst werden müssen. Die Kapitel (und damit auch die Aufgaben) könnten sich grob an historischen Ereignissen der Brauerei orientieren und in chronologischer Reihenfolge durchgespielt werden.

Starten würde das Spiel also mit einer Art Szenenauswahl:
![image](https://user-images.githubusercontent.com/22897030/41656424-2ebb1bb6-7490-11e8-8d1b-45c1b24ccb0e.png)

**Comment by ChrisOKay** [comment]
Eine chronologisches Durcharbeiten ist sicherlich ein schöner roter Faden. Das Interface dazu ist auch ok. Für einen ersten Test absolut ausreichend. Vielleicht fällt uns beim zuordnen der Minispiele ja dann noch eine Alternative ein, aber wenn nicht, würde das als finales Ziel sicherlich auch gut sein.

**Comment by t-karcher** [comment]
Alternativ könnten wir die Minispiele auch einfach in loser Folge a lá Yeti Sports präsentieren - also völlig ohne Rahmenhandlung oder zeitliche Einordnung, zusammengehalten nur vom Thema Bier / Brauerei. Hätte den Vorteil, dass wir sie nicht zwangshaft in irgendetwas "einbetten" müssen, und dass unser Spiel schon mit mit ersten Minispiel einen "Minimum Viable Product"-Status erreicht.

**Comment by t-karcher** [comment]
Hab nochmal darüber nachgedacht: Die aufeinanderfolgenden Kapitel gefallen mir doch besser als die lose Folge von Minipuzzlen. Vielleicht sogar (vereinfacht!) wie bei Tiny Bang Story, also mit Bruchstücken aus verschiedenen Teilen der Geschichte, welche am Schluß zusammengesetzt werden müssen?

**Comment by ChrisOKay** [comment]
Die Hauptaufgabe sind meiner Meinung nach die Spiele selbst. Je nachdem, wie viele wir implementieren wollen / fertigstellen können wir darauf basierend noch immer entscheiden, wie der "allgemeine Rahmen" aussehen soll und entsprechend die Grafiken zu den Spielen anpassen.

**Comment by t-karcher** [comment]
Beim Durchlesen deiner neuesten Minispiel-Ideen kam mir gerade noch ein Gedanke: Man könnte *die gleichen* Minispiele auch für verschiedene Epochen abwandeln. Zum Beispiel ein Hoffest veranstalten zu Beginn des Jahrhunderts, eines in den 60ern und eines heute. Vorteil: Die Spielmechanik könnte relativ gleich bleiben, und man hätte trotzdem mehr Spaß als nur mit einem Minispiel.

**Comment by ChrisOKay** [comment]
Ja, das verringert außerdem den Entwicklungsaufwand etwas ;)
Wann beginnen wir den mit dem Filtern? Erst, wenn 50+ erreicht sind? Allmählich werden meine Ideen.... etwas aus der Luft gegriffen. Hatte mir überlegt, dass wir (jeder für sich) jede Idee gegen jede andere antreten lassen, um daraus ein Ranking zu erstellen, allerdings sind es bereits jetzt 36 Spiele, d.h. es sind 35+34+33+... = 36*35/2 = 630 Vergleiche sind, was.... irgendwie zeitraubend und ineffizient klingt. Hast du eine gute Idee hierzu?

**Comment by t-karcher** [comment]
Kommt ihr am Wochenende auch an den Lohberg? Dann könnten wir das dort diskutieren und vielleicht auch schon ein bisschen programmieren!

**Comment by ChrisOKay** [comment]
Ja, wir sind aber nur Sonntags da. Zum Diskutieren wird es reichen, zum Programmieren wahrscheinlich nicht.
